cmd/verify/verify.go

```diff
@@ -146,18 +158,24 @@ func main() {
 
 			stream := io.TeeReader(c4ghr, md5hash)
 
-			if _, err := io.Copy(sha256hash, stream); err != nil {
+			if file.DecryptedSize, err = io.Copy(sha256hash, stream); err != nil {
 				log.Error(err)
 				continue
 			}
 
+			file.Checksum = archiveFileHash
+			file.DecryptedChecksum = sha256hash
+
 			//nolint:nestif
 			if message.ReVerify == nil || !*message.ReVerify {
-				log.Debug("Mark completed")
+				log.Debug("will run markcompleted")
```

Comment:
This seems redundant

Reply:
Addressed in 846fb9a0ba5043b88fa5486d55b38c2419c0295e.